plugins/catalog-backend-module-gitlab/src/providers/GitlabOrgDiscoveryEntityProvider.ts

```diff
@@ -781,4 +777,13 @@ export class GitlabOrgDiscoveryEntityProvider implements EntityProvider {
       entity,
     ) as Entity;
   }
+
+  private getRelations(config: any) {
+    return [
+      'DIRECT',
+      ...(config.allowInherited ? ['INHERITED'] : []),
+      ...(config.allowSharedFromGroups ? ['SHARED_FROM_GROUPS'] : []),
+      ...(config.allowDescendants ? ['DESCENDANTS'] : []),
```

Comment:
Is there a specific reason for restricting what types of relations can be accepted? Could we receive the array of relations as config, allowing people to pass a combination rather than creating a configuration for each possible value?

Reply:
Hi @camilaibs,  

There is no specific reason and this is actually a great idea!  :D

The GitLab GroupMemberRelation GraphQL query supports including direct, descendant, inherited, and members from shared groups. Currently, the `GitLabOrgDiscoveryEntityProvider` defaults to including direct group members and also provides the `allowInherited` option to include members from subgroups in the ingestion process.

Based on your feedback, I've introduced a new `relations` array configuration option. To ensure backward compatibility, the existing `allowInherited` option has been kept but marked as deprecated. Is this ok? 
Please let me know your thoughts!

Also, some of the CI jobs have failed but I cannot see the logs and don't understand what happened. Should I do anything?

Reply:
All changes look good to me, thanks for explaining 🙂